lighthouse-core/report/html/report-styles.css

```diff
@@ -786,12 +786,13 @@
   border: 0;
 }
 
-.lh-scores-header .lh-gauge__wrapper,
-.lh-scores-header .lh-gauge--pwa__wrapper {
+.lh-scores-header .lh-gauge__wrapper {
   margin: 0 4px;
```

Comment:
sorry one last thing, is there a reason we picked margin at all? why not just go `padding: 0 4px;` :)

Reply:
I assumed there was a reason, but I have no idea.  My CSS gut says this _is margin_ not padding, but I have no strong opinion at all.  Might be better to simplify.

Reply:
looks like it was added [here](https://github.com/GoogleChrome/lighthouse/pull/5706/files#diff-f8838d243bb796c2800f4eceab80b7ddR724) @paulirish any memory of why we might not want to just go with padding here now?

Reply:
padding wfm!

Reply:
there was no border before, so it might have just not mattered at the time

Reply:
changed to padding :smile: